webapp/tests/karma/unit/services/format-date.js

```diff
@@ -121,6 +121,8 @@ describe('FormatDate service', function() {
       var actual = service.relative(moment(), { withoutTime: true });
       chai.expect(actual).to.equal('pretty soon');
       chai.expect(translateInstant.args[0][0]).to.equal('today');
+      actual = service.relative(moment().startOf('day/'), { withoutTime: true });
+      chai.expect(actual).to.equal('pretty soon');
```

Comment:
This is a useful assertion but it's not testing what the test description says it is. Make a new test to test this with a description something like... 'returns "today" when between midnight and now'